### PR TITLE
WebDriver: Allow running dinghy in automation mode

### LINKED ISSUE #7: Support automation mode for WebDriver
This would need accepting an `--automation` CLI flag, which:

- Uses an ephemeral `WebKitWebContext`.
- Sets the `WebKitWebView::is-controlled-by-automation` property.
- If no URL is provided in the command line, loads `about:blank` by default.

For inspiration, check the changes in https://github.com/Igalia/dyz/commit/19d512fbcd3892491de3adbbbef030ddb807d294

### PR DESCRIPTION
**Pull request by aperezdc**
This is needed to enable remotely controlling a Dinghy instance by using WebDriver.

Fixes #7

### CODE REVIEW

File: dinghy.c [hunk 1]
```diff
@@ -29,6 +29,7 @@ static struct {
     gboolean log_console;
     gboolean file_uri_strict;
     gdouble  scale_factor;
+    gdouble  automation_mode;
```
Comment (carlosgcampos) [inline dinghy.c]: This should be gboolean

File: dinghy.c [hunk 2]
```diff
@@ -274,6 +278,9 @@ on_handle_local_options (GApplication *application,
                                          handler);
     }
 
+    if (s_options.automation_mode)
+        dy_launcher_enable_automation (DY_LAUNCHER (application));
+
     dy_launcher_set_home_uri (DY_LAUNCHER (application), utf8_uri);
```
Comment (carlosgcampos) [inline dinghy.c:288]: In case of automation mode it's better to ignore any homepage and always load about:blank at startup.

**Review by carlosgcampos — CHANGES_REQUESTED** [review]
LGTM. I only have two comments.

File: dinghy.c (hunk 1 above)
Comment (aperezdc) [reply dinghy.c]: :man_facepalming: 

Good catch!

File: dinghy.c (hunk 2 above)
Comment (aperezdc) [reply dinghy.c:288]: I'll make it load `about:blank` when automation is enabled, thanks for the tip.

**Pull request merged**